shared-helpers/src/locales/general.json

```diff
@@ -936,19 +936,22 @@
   "listings.reservedCommunityTitleDefault": "Reserved Building",
   "listings.reservedCommunityType": "Reserved Community Type",
   "listings.reservedCommunityTypeDescriptions.developmentalDisability": "People with Developmental Disabilities",
+  "listings.reservedCommunityTypeDescriptions.seniorVeterans": "Reserved for seniors 62+ years old who have served in the US military and who were honorably discharged.",
```

Comment:
Public site already adds `Reserved for` to the beginning of every description. Should remove from this one and all other translations

Reply:
nice catch! updated